libs/openchallenges/config/src/lib/app.config.ts

```diff
@@ -20,6 +20,7 @@ export interface AppConfig {
   ssrApiUrl: string;
   termsOfUseUrl: string;
   apiDocsUrl: string;
+  showAnnouncement: string;
```

Comment:
Can this type be set to boolean and do the conversion type when the config file is read? It's actually not ideal if the conversion type / value check needs to be done in the Angular components.

Reply:
If that can't be done easily, let's just remove the banner and the feature flag for the sake of time.